tests/proxy/get_remote.t

```diff
@@ -2,4 +2,4 @@
   $ cd ${TESTTMP}
 
   $ curl -s http://localhost:8002/remote
-  http://localhost:8001 (no-eol)
+  {"Http":"http://localhost:8001"} (no-eol)
```

Comment:
Will this break the UI?

Reply:
So this means UI does not work if there is no http remote? Makes some sense. How would it authenticate...
So for now, yes, error if there is none. As long as there is no concrete use case of exposing ssh remote (like there is with http) we don't need an API.